src/odemis/gui/cont/tabs.py

```diff
@@ -1870,6 +1870,22 @@ def display_new_data(self, filename, data, extend=False):
         # TODO: Move viewport related code to ViewPortController
         # TODO: to support multiple (types of) streams (eg, AR+Spec+Spec), do
         # this every time the streams are hidden/displayed/removed.
+
+        # handle display when both data types are present.
```

Comment:
Let's add a "TODO: remove once it's possible to visualise both stream simultaneously".